apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-preview-icon.tsx

```diff
@@ -35,17 +35,19 @@ interface ResourcePreviewImageProps {
  */
 
 const lookupTable: { [K in ResourceType]: JSX.Element } = {
-    [ResourceType.audioItem]: <VolumeUpIcon fontSize="inherit" />,
-    [ResourceType.bibliographicReference]: <CollectionsBookmarkIcon fontSize="inherit" />,
-    [ResourceType.book]: <AutoStoriesIcon fontSize="inherit" />,
-    [ResourceType.mediaItem]: <SubscriptionsIcon fontSize="inherit" />,
-    [ResourceType.photograph]: <PhotoIcon fontSize="inherit" />,
-    [ResourceType.song]: <AudiotrackIcon fontSize="inherit" />,
-    [ResourceType.spatialFeature]: <PlaceIcon fontSize="inherit" />,
-    [ResourceType.term]: <LanguageIcon fontSize="inherit" />,
-    [ResourceType.video]: <VideocamIcon fontSize="inherit" />,
-    [ResourceType.vocabularyList]: <ListIcon fontSize="inherit" />,
-    [ResourceType.playlist]: <PlaylistPlayIcon fontSize="inherit" />,
+    [ResourceType.audioItem]: <VolumeUpIcon fontSize="inherit" color="primary" />,
+    [ResourceType.bibliographicReference]: (
+        <CollectionsBookmarkIcon fontSize="inherit" color="primary" />
+    ),
+    [ResourceType.book]: <AutoStoriesIcon fontSize="inherit" color="primary" />,
```

Comment:
If the colour will always be the same for all of these, perhaps there is a pattern where you only need to declare the colour once (outside of this file?)

Reply:
I should be able to fix this on a near-future refactor that is on my radar anyway.